packages/components/tests/dummy/app/templates/components/dropdown.hbs

```diff
@@ -671,7 +669,40 @@
           <Hds::Dropdown::ListItem
             @route="components.dropdown"
             @icon={{if (eq color "critical") "trash" "settings"}}
-            @text="{{state}} with a long multi-line text"
+            @text="{{state}} with a longer text string that may wrap since max-width is defined on the container"
```

Comment:
when I read this, it was confusing for me (I thought the container was an external wrapper; here it's actually the list item itself, so maybe we should say something about the fact that the item has a max width?
```suggestion
            @text="{{state}} with a longer text string that may wrap since the list item has a max-width property applied to it"
```

Reply:
that's not technically accurate; it's the container (the `<ul>` that has the sizing: 

```scss
  .hds-dropdown-list {
    ...
    min-width: 200px;
    max-width: 400px;
  }
```

But, we're still working on the proportions and converting these to relative units, so I think we should fine-tune the wording in the last pass of the docs. WDYT?